test/dynamo/test_verify_correctness.py

```diff
@@ -100,8 +100,11 @@ def compiler_fn(graph, example_inputs):
         r3 = opt_fn(a, (b, c), d)
 
         self.assertIsNotNone(r1)
-        self.assertTrue(same(r1, r2))
-        self.assertTrue(same(r1, r3))
```

Comment:
Why don't you just assert `same(r2, r3)` and call it a day lol

Reply:
we care about r1 as well